### DISCUSSION TITLE
Student application

### DISCUSSION THREAD

**Discussion by MichaelPaulRoth** [community/community#52041] [GitHub Education]
Good morning everyone, I am sorry to bother you (I hope I am making this post in the correct place). I am a student doing a Post Graduate Diploma at Varsity College in Cape Town South Africa and the course demands that we use Python extensively. My lecturer recommended that I sign up to a student GitHub account so that I can use the GitHub copilot extension in VS Code to help me code and comment on code (to help me understand what the code the lecturer has given us does).  However, my application continues to be declined (sometimes it says that my name on my Github Profile doesn't match the document I'm sending in or that my profile needs to be completed (social media links and bios (I have added as much information to my profile as possible). I am in desperate need of GitHub copilot and I am quite stuck without it (our lecturer has shown us its power during classes). If there is any information I can provide to help my application, please let me know.

Kind regards, 
Michael

**Comment by Ntshangase** [comment]
Hi @MichaelPaulRoth 

This post is definitely in the correct place. To help us better understand your problem it's recommended that you send a screenshot of the rejection reason.

- GitHub uses your GitHub profile to cross reference the documents you submitted during application and it easier if you have the name on the document match the one on your GitHub profile. 
- having some activity in your GitHub account is also helpful, maybe creating a repository would help.

**Reply by MichaelPaulRoth** [reply]
Good afternoon, originally it was saying that i needed to complete my profile set up (which i did with the bio and social media links etc). 

Then it said that i should take a picture instead of uploading a file (i uploaded my acceptance letter) because uploaded files can sometimes not be genuine.

Then i took a picture of my student card (which doesn't have a date because it is meant to last for your whole student career and is usually only replaced if damaged or lost).

It then said that the name on my profile needs to match the name on the documents i am sending in (i originally created my github account with the name of H3LLSASSASS1N a long time ago (i used to be a gamer and didnt think i would go into a career field of study program that required the use of extensive coding)). I have changed both my user name and my name on my profile to the same thing (my real name) but i still get this "your name on the submitted documents needs to match the name on your profile" error. I will attach screen shots of the emails i got once my application was submitted. (I screen shot the images and then whatsapped them to myself).

![WhatsApp Image 2023-04-05 at 15 20 21](https://user-images.githubusercontent.com/89579758/230093635-5b62fb5a-1527-4d28-b91d-39655681a0dd.jpeg)
![WhatsApp Image 2023-04-05 at 15 20 23](https://user-images.githubusercontent.com/89579758/230093650-9c00d86a-3402-4d4e-8e95-6046d87c8751.jpeg)
![WhatsApp Image 2023-04-05 at 15 20 24](https://user-images.githubusercontent.com/89579758/230093658-5661e9a5-f9da-4673-bc54-f31f4586c269.jpeg)
![WhatsApp Image 2023-04-05 at 15 20 25](https://user-images.githubusercontent.com/89579758/230093664-30c62dd4-c197-40e5-be56-17c505b2e477.jpeg)

**Reply by samus-aran** [reply]
Great suggestions @Ntshangase 🎉 
I would just add when asking people to update their profile to let them know to log out and log back in again before re-applying to ensure the application get's the latest profile updates 👍

**Comment by Ntshangase** [comment]
Your profile seems complete to me,
The ID card is suppose to be dated to prove that you are registered for this year, Maybe try using your proof of registration and see if that works.

**Reply by MichaelPaulRoth** [reply]
I got an offer letter but i got that in November/December 2022 which it is dated, has all the course information and my name and the start date of the course. I also have a proof of admission letter but it doesn't have a date on it.

**Reply by Ntshangase** [reply]
I think it will be easier if you have something that is dated with this years date.

**Reply by MichaelPaulRoth** [reply]
The problem is, i don't and that is all the official documentation i have from the university.

**Comment by samus-aran** [comment]
Hi @MichaelPaulRoth 👋 

Let's see how we can assist.

1. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

From your comments you have a Student ID, then I suggest you stick with this. Although we do prefer Dated we understand that Student ID may not all have that.

2. The application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

3. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
  1. If you can, print off your evidence and take a photo of this copy.
  2. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know how you get on 👍

**Comment by dennybill** [comment]
In the meantime, there are other resources you can use to improve your Python skills. There are many online tutorials, courses, [Yowhatsapp](https://yowaapp.download/) groups and forums that can help you learn Python and gain a better understanding of the code your lecturer has given you. You can also consider using other code editors or IDEs that have similar features to GitHub Copilot, such as PyCharm or Jupyter Notebook.
